packages/avatar-group/src/vaadin-avatar-group.js

```diff
@@ -12,7 +12,7 @@ import { Debouncer } from '@polymer/polymer/lib/utils/debounce.js';
 import { IronA11yAnnouncer } from '@polymer/iron-a11y-announcer/iron-a11y-announcer.js';
 import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { IronResizableBehavior } from '@polymer/iron-resizable-behavior/iron-resizable-behavior.js';
-import { ElementMixin } from '@vaadin/vaadin-element-mixin/vaadin-element-mixin.js';
+import { ElementMixin } from '@vaadin/component-base';
```

Comment:
Let's use direct imports as we do with other mixins:

```suggestion
import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
```

Please update `.d.ts` file accordingly.